base/strings/basic.jl

```diff
@@ -403,6 +403,31 @@ julia> lowercase("STRINGS AND THINGS")
 lowercase(s::AbstractString) = map(lowercase, s)
 
 """
+    titlecase(s::AbstractString)
+
```

Comment:
You'll also need to add a placeholder to `doc/stdlib/strings.rst` for `titlecase` to be included in the manual, and re-run `julia doc/genstdlib.jl`